demos/projects/ESPRESSIF/aziotkit/components/azure-iot-kit-sensors/src/sensor_manager.h

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#ifndef SENSOR_MANAGER_H
+#define SENSOR_MANAGER_H
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+    typedef void (*button_cb)(void *);
+
+    void initialize_sensors();

```

Comment:
I see everything dumped here ... should we have led also here and remove all the other headers?